data/json/items/comestibles/carnivore.json

```diff
@@ -236,7 +236,6 @@
     "name": { "str_sp": "human flesh" },
     "description": "A chunk of raw meat, butchered from a human body.",
     "material": [ "hflesh" ],
-    "vitamins": [ [ "calcium", 2 ], [ "iron", 12 ], [ "human_flesh_vitamin", 100 ] ],
```

Comment:
Is this an intentional change?

Reply:
No clue how that got there, good catch

Reply:
There's a few other oddities to do with cannibal stuff you probably want to check the files changes tab

Reply:
alright